thundra/wrappers/aws_lambda/lambda_executor.py

```diff
@@ -132,48 +133,14 @@ def process_api_gw_response(execution_context):
         pass
 
 
-def set_start_time(execution_context):
-    if not execution_context.start_timestamp:
-        execution_context.start_timestamp = int(time.time() * 1000)
-
-
-def set_end_time(execution_context):
-    if not execution_context.finish_timestamp:
-        execution_context.finish_timestamp = int(time.time() * 1000)
-
-
 def start_invocation(plugin_context, execution_context):
-    set_start_time(execution_context)
+    wrapper_utils.set_start_time(execution_context)
```

Comment:
These `wrapper_utils.set_start_time` and `wrapper_utils.set_end_time` methods seems like being called in every possible place. Couldn't we instead set `start_time` and `end_time` properties in one common place?

Reply:
For example, [this](https://github.com/thundra-io/thundra-lambda-agent-nodejs/blob/2713a090c3d71c3d2ff6d188fc343494f6ed39d7/src/wrappers/WebWrapperUtils.ts#L104) is such common place that I've mentioned above for the Node.js agent.